app/download/page.tsx

```diff
@@ -33,6 +34,9 @@ export default function Download() {
       <table className="w-full border-collapse bg-white text-left text-sm text-gray-500">
         <thead className="bg-gray-50">
           <tr>
+            <th scope="col" className="px-6 py-4 font-medium text-gray-900">
+              hoge
+            </th>
```

Comment:
```suggestion
            <th scope="col" className="px-6 py-4 font-medium text-gray-900">
            </th>
```
IMO, the value is not needed

Reply:
Deleted value.
Fixed according to biome.